src/renderer/components/deposit/withdraw/Withdraw.tsx

```diff
@@ -325,6 +329,10 @@ export const Withdraw: React.FC<Props> = ({
 
       <Styled.AssetContainer>
         <Styled.AssetIcon asset={asset} network={network} />
+        <Styled.Label nowrap>
+          <Styled.TickerLabel className="ticker">{asset.ticker}</Styled.TickerLabel>
+          <Styled.TickerLabel className="small">{asset.chain}</Styled.TickerLabel>
```

Comment:
Pls provide `Styled.TickerLabel` + `Styled.ChainLabel`. Both are very similar styled components, but we should not to override it by using `className` in other components. Imagine we change ".small" to ".chain" or so, all overridden styles will be lost.

Reply:
Let do it with another PR after @sarawut11 has finished #1136. _Approving now..._

Reply:
Just fixed and pushed

Reply:
_It seems we can use `AssetData` here. Anyway, this change is not needed yet..._